snakefmt/formatter.py

```diff
@@ -6,10 +6,12 @@
 from snakefmt import DEFAULT_LINE_LENGTH
 from snakefmt.exceptions import InvalidPython, InvalidParameterSyntax
 from snakefmt.parser.parser import Parser
+from snakefmt.parser.grammar import SnakeRule
 from snakefmt.parser.syntax import (
     Parameter,
     ParameterSyntax,
     SingleParam,
+    SingleNumericParam,
     TAB,
```

Comment:
I think we might need to create a very small class (enum) for things like tabs and newlines. They're used everywhere and if we want to change one we have to change it in a lot of places.  
I would suggest something like

```py
from typing import NamedTuple

class Ascii(NamedTuple):
    TAB = " " * 4
    NL = "\n"

>>> Ascii.TAB
'    '
>>> Ascii.NL
'\n'
```

Reply:
Would also make writing tests a lot easier

Reply:
Does `TAB` not achieve this?
But yes why not, happy with that enum

Happy with you adding those two changes in a commit and then merging to dev

Reply:
Yeah, `TAB` achieves that. It just triggered me to think that adding one for newlines as well would be good (and in case any other special characters pop up in the future)